azext_iot/_help.py

```diff
@@ -912,3 +912,253 @@
+        --interface {plug_and_play_interface} --consumer-group {consumer_group_name}
+    - name: Monitor digital twin events within current session.
+      text: >
+        az iot dt monitor-events --hub-name {iothub_name} --device-id {device_id} --source public
+        --interface {plug_and_play_interface} --consumer-group {consumer_group_name}
+    - name: Monitor digital twin events and see all message properties.
+      text: >
+        az iot dt monitor-events --login {iothub_cs} --device-id {device_id}
+        --interface {plug_and_play_interface} --consumer-group {consumer_group_name}
+        --properties all --source device
+"""
+
+helps['iot dt update-property'] = """
+    type: command
+    short-summary: Update an IoT Plug and Play device interfaces read-write property.
+    examples:
+    - name: Update an IoT Plug and Play device interfaces read-write property.
+      text: >
+        az iot dt update-property --login {iothub_cs} --device-id {device_id}
+        --interface-payload {payload}
+    - name: Update an IoT Plug and Play device interfaces read-write property within current session.
+      text: >
+        az iot dt update-property --hub-name {iothub_name} --device-id {device_id}
+        --interface-payload {payload}
+"""
+
+helps['iot pnp'] = """
+    type: group
+    short-summary: Manage entities of an IoT Plug and Play model repository.
+"""
+
+helps['iot pnp interface'] = """
+    type: group
+    short-summary: Manage interfaces in an IoT Plug and Play model repository.
+"""
+
+helps['iot pnp interface publish'] = """
+    type: command
+    short-summary: Publish an interface.
+    examples:
+    - name: Publish an interface.
+      text: >
+        az iot pnp interface publish -r {pnp_repository} --interface {plug_and_play_interface_id}
+"""
+
+helps['iot pnp interface create'] = """
+    type: command
+    short-summary: Create an interface in a repository.
+    examples:
+    - name: Create an interface in a repository.
+      text: >
+        az iot pnp interface create --def {plug_and_play_interface_file_path} -r {pnp_repository}
+"""
+
+helps['iot pnp interface update'] = """
+    type: command
+    short-summary: Update an interface in a repository.
+    examples:
+    - name: Update an interface in a repository.
+      text: >
+        az iot pnp interface update --def {updated_plug_and_play_interface_file_path} -r {pnp_repository}
+"""
+
+helps['iot pnp interface list'] = """
+    type: command
+    short-summary: List all interfaces.
+    examples:
+    - name: List all private repository's interfaces.
+      text: >
+        az iot pnp interface list -r {pnp_repository}
+    - name: List all public interfaces.
+      text: >
+        az iot pnp interface list
+"""
+
+helps['iot pnp interface show'] = """
+    type: command
+    short-summary: Get the details of an interface.
+    examples:
+    - name: Get the details of a private repository interface.
```

Comment:
For list/show of interface or cm, there is specific reference to public or private repos. It shows different usage of the command for public vs private. Contrasting with create/update/publish there is just repo, without consideration of public/private. Is this intended?

Reply:
Yes, create/update/publish is applicable for private repo only